dbmanager/dbmanager.go

```diff
@@ -0,0 +1,116 @@
+package dbmanager
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	_ "github.com/lib/pq" //db driver for postgres
+	"golang.org/x/crypto/bcrypt"
+
+)
+
+//////// QUERIES
+const (
+	userExistsQuery  = `SELECT uid from users WHERE email = $1`
+	getUserInfoQuery = `SELECT * from users WHERE email = $1`
+	getUserPassQuery = `SELECT password from users WHERE email = $1`
+	createUserQuery  = `INSERT INTO users(uid,email,password) VALUES (DEFAULT, $1 , $2);`
+)
```

Comment:
I'm not opposed to storing all queries as constants here, but it leads to a lot of scrolling back & forth from where the values are used. Seems ok for now though.

Reply:
@pierrebeaucamp would you prefer to see the raw query placed in each location where used? Easy change, definitely don't mind making it.

Reply:
It's up to you. Personally, I'd probably place it closer to where it's used, but there are probably some advantages and disadvantages either way.

I was mainly wondering if there was a strong reason to store them outside their respective function body.